Frontend/ui-driver/src/Screens/ReferralScreen/Controller.purs

```diff
@@ -159,10 +162,15 @@ eval (UpdateLeaderBoard (LeaderBoardRes leaderBoardRes)) state = do
         selectedWeek = case last pastWeeks of
                         Just week -> week
                         Nothing -> state.props.selectedWeek
+    _ <- pure $ setRefreshing "2000034" false
     continue newState{ props{ days = pastDates, weeks = pastWeeks, selectedDay = selectedDay, selectedWeek = selectedWeek } }
-  else continue newState
+  else do 
+    _ <- pure $ setRefreshing "2000034" false
```

Comment:
can you move this out of `if-else`

Reply:
moved the statement out of the `if-else` block